src/EditorFeatures/Core/Implementation/ForegroundNotification/ForegroundNotificationService.cs

```diff
@@ -31,13 +30,14 @@ internal class ForegroundNotificationService : ForegroundThreadAffinitizedObject
         private int _lastProcessedTimeInMS;
 
         [ImportingConstructor]
-        public ForegroundNotificationService()
+        public ForegroundNotificationService(IThreadingContext threadingContext)
```

Comment:
Missing `[Obsolete]` attribute?

Reply:
➡️ For this one, the only usage is in `ForegroundNotificationServiceTests`. Prior to preparing this PR, I actually updated the test to use the MEF catalog instead (and mark this constructor obsolete), but in the end I eliminated it as a source of just more noise in a fairly overwhelming PR.